api/src/OpenTelemetry/Internal/Logging/Types.hs

```diff
@@ -92,7 +170,7 @@ data LogRecord body = LogRecord
   --
   -- In the contexts where severity participates in less-than / greater-than comparisons SeverityNumber field should be used.
   -- SeverityNumber can be compared to another SeverityNumber or to numbers in the 1..24 range (or to the corresponding short names).
-  , logRecordBody :: body
+  , logRecordBody :: AnyValue
```

Comment:
Replaced the `body` type variable with `AnyValue` so that if logs come in from different sources with different body types, processors and exporters can still process them. `body` is defined [here in the spec](https://opentelemetry.io/docs/specs/otel/logs/data-model/#field-body).